libyang: absent dependency on libxxhash

### Package Name

libyang

### Maintainer

Jakov Smolic <jakov.smolic@sartura.hr>

### OpenWrt Version

SNAPSHOT

### OpenWrt Target/Subtarget

x86/64

### Steps to Reproduce

1. Compile package

### Actual Behaviour

Fails on apk check:
```
Package libyang is missing dependencies for the following libraries:
libxxhash.so.0
make[2]: *** [Makefile:70: /home/user/LEDE/openwrt/bin/packages/x86_64/packages/libyang-3.13.6-r1.apk] Error 1
```

Fix:
```diff
diff --git a/libs/libyang/Makefile b/libs/libyang/Makefile
index 03578a415..7909e5056 100644
--- a/libs/libyang/Makefile
+++ b/libs/libyang/Makefile
@@ -31,7 +31,7 @@ define Package/libyang
   CATEGORY:=Libraries
   TITLE:=YANG data modeling language library
   URL:=https://github.com/CESNET/libyang
-  DEPENDS:=+libpcre2 +libpthread
+  DEPENDS:=+libpcre2 +libpthread +libxxhash
 endef

 define Package/yanglint
```

### Confirmation Checklist

- [x] The package is maintained in this repository.
- [x] I understand that issues related to [the base OpenWrt repository](https://github.com/openwrt/openwrt/issues) or [LuCI repository](https://github.com/openwrt/luci/issues) will be closed.
- [x] I am reporting an issue for OpenWrt, not an unsupported fork.

--- a/libs/libyang/Makefile
+++ b/libs/libyang/Makefile
@@ -31,7 +31,7 @@ define Package/libyang
   CATEGORY:=Libraries
   TITLE:=YANG data modeling language library
   URL:=https://github.com/CESNET/libyang
-  DEPENDS:=+libpcre2 +libpthread
+  DEPENDS:=+libpcre2 +libpthread +libxxhash
 endef
 
 define Package/yanglint

